### PR TITLE
lando: add page

### PR DESCRIPTION
**Pull request by ZachEngstrom**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/lando.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# lando
+
+> Local development environment and DevOps tool built on Docker.
+> More information: <https://docs.lando.dev/cli/>.
+
+- Initialize code for use with lando:
```
Comment (kbdharun) [inline pages/common/lando.md:6]:
```suggestion
- Initialize code for use with `lando`:
```

File: pages/common/lando.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`lando init`
+
+- Print info about your app:
```
Comment (kbdharun) [inline pages/common/lando.md:10]:
```suggestion
- Print information about your app:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by ZachEngstrom** [comment]
> Welcome to tldr and Thanks for your contribution. I have some minor suggestions for this page.

@kbdharun - I've committed your suggestions. Thank you!

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**